ZPastel.Test/Builders/FilterPastelResourceBuilder.cs

```diff
@@ -0,0 +1,46 @@
+﻿using ZPastel.API.Resources;
+
+namespace ZPastel.Test.Builders
+{
+    public class FilterPastelResourceBuilder
+    {
+        private readonly PastelFilterResource pastelFilterResource;
+
+        public FilterPastelResourceBuilder()
+        {
+            pastelFilterResource = new PastelFilterResource();
+        }
+
+        public FilterPastelResourceBuilder WithDefaultValues()
+        {
+            pastelFilterResource.Take = 25;
+            pastelFilterResource.Skip = 0;
+            pastelFilterResource.Name = "Pastel";
+
+            return this;
+        }
+
+        public FilterPastelResourceBuilder WithName(string name)
+        {
+            pastelFilterResource.Name = name;
+
+            return this;
+        }
+
+        public FilterPastelResourceBuilder WithTake(int take)
+        {
+            pastelFilterResource.Take = take;
+
+            return this;
+        }
+
+        public FilterPastelResourceBuilder WithSkip(int skip)
```

Comment:
This pagination methods are not needed

Reply:
Removed pagination methods and properties